src/NServiceBus.AcceptanceTests/Routing/When_configuring_routing.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+
+    public class When_configuring_routing : NServiceBusAcceptanceTest
```

Comment:
@weralabaj I know that @timbussmann loves ATTs ;) I personally find it a bit overkill to test a method on an extension method or a class with an acceptance test. I would have used a simple unit test instead. But I'll let @timbussmann decide if he thinks it is really necessary I won't object and run with it.